slack_sdk/oauth/installation_store/models/installation.py

```diff
@@ -64,9 +79,13 @@ def __init__(
             self.user_scopes = user_scopes
 
         self.incoming_webhook_url = incoming_webhook_url
+        self.incoming_webhook_channel = incoming_webhook_channel
         self.incoming_webhook_channel_id = incoming_webhook_channel_id
         self.incoming_webhook_configuration_url = incoming_webhook_configuration_url
 
+        self.is_enterprise_install = is_enterprise_install
```

Comment:
SQLite 3 does not have boolean data type internally but using boolean in DDL should work. https://sqlite.org/datatype3.html

Reply:
I'm not sure what I should change for this

Reply:
sorry, I placed this comment in a wrong line